### PR TITLE
[W5.11][F11-A2] Zhang Yijiong

### PR DESCRIPTION
**Pull request by Zhang-Y-J**
Second enhancement, add sort command to display persons in lexicographic order

### CODE REVIEW

**Comment by niloc94** [comment]
Travis Bot failed because there are whitespaces in the actual output.
Try using a text editor to add the space to the expected output. I had issues with adding the space in IntelliJ (not sure if you are facing the same problem), but manage to solve it with a text editor.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+
```
Comment (karrui) [inline src/seedu/addressbook/commands/SortCommand.java:3]: Extra line

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allSortPersons = addressBook.getAllPersons().immutableSortedListView();
+        return new CommandResult(getMessageForSortedPersonShownSummary(allSortPersons), allSortPersons);
+    }
+}
+
+
```
Comment (karrui) [inline src/seedu/addressbook/commands/SortCommand.java:26]: Extra lines. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book in ascending order lexicographically and displays them.\n"
+            + "Example: " + COMMAND_WORD;
+
+
```
Comment (karrui) [inline src/seedu/addressbook/commands/SortCommand.java:18]: Extra line

File: doc/UserGuide.md [hunk 4]
```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons : `sort`
+Shows a list of persons sorted lexicographically in the address book.<br>
```
Comment (karrui) [inline doc/UserGuide.md:57]: Your sort command seems to save the sorted order (i.e. persistent sorting). Might be a good idea to tell the user that this command irreversibly sorts the list.

**Review by karrui — COMMENTED** [review]
Some comments added. Please close the PR after reading comments.

**Comment by Zhang-Y-J** [comment]
Comments read.

**Pull request closed without merging**